open-metadata-implementation/adapters/open-connectors/governance-daemon-connectors/open-lineage-connectors/open-lineage-janus-connector/src/main/java/org/odpi/openmetadata/openconnectors/governancedaemonconnectors/openlineageconnectors/janusconnector/graph/LineageGraphConnectorHelper.java

```diff
@@ -119,23 +115,32 @@ LineageVerticesAndEdges ultimateSource(String guid, boolean includeProcesses) {
         return getCondensedLineage(guid, g, sourceGraph, getLineageVertices(sourcesList), SOURCE_CONDENSATION, includeProcesses);
     }
 
+ /*
```

Comment:
In case we need it, we can take the old query from git history, so deleting it is more fitting